src/main/java/mil/dds/anet/config/AssetConfig.java

```diff
@@ -15,6 +15,7 @@ public class AssetConfig implements WebMvcConfigurer {
   public static final String ASSETS_PATH = "/assets/**";
   public static final String IMAGERY_PATH = "/imagery/**";
   public static final String DATA_PATH = "/data/**";
+  public static final String GRAPHQL_WEB_SERVICE = "/graphqlWebService/**";
```

Comment:
This does not belong in AssetConfig, the web service is not a static asset. Instead, move it to GraphQLWebService as:
```java
public static final String GRAPHQL_WEB_SERVICE = "/graphqlWebService";
```
and use that constant in places where you currently hard-code `"/graphqlWebService"`.